design-tokens/properties/status/base.json

```diff
@@ -2,7 +2,7 @@
   "status": {
     "primary": {
       "color": {
-        "light": { "value": "{color.base.sky-blue.500.value}" },
+        "light": { "value": "{color.base.sky-blue.100.value}" },
```

Comment:
I did pull from `develop` branch, this PR was outdated, but I don't remember this change from `500` to `100`. Do you remember?